src/stylesheets/layouts/_factors.scss

```diff
@@ -1,22 +1,21 @@
 .key-factors {
-  @include flex;
   background-color: $color-mint;
-  margin-top: 1.5rem;
+  margin-top: 15px;
```

Comment:
If you already have a `rem` system (assuming you're using 62.5% to get base of 1rem = 10px), why move to pixels for layout?

Reply:
I wanted to streamline the units of measurement and (from my understanding 😐 ) pixels were recommended for vertical units. Thanks for the clarification!! Refactored:
- media queries --> em
- units --> rem
- line-heights --> unitless